pkgs/development/libraries/libxkbcommon/default.nix

```diff
@@ -21,8 +31,7 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" "doc" ];
 
-  nativeBuildInputs = [ meson ninja pkg-config yacc doxygen ]
-    ++ lib.optional withWaylandSupport wayland;
```

Comment:
I think we want to keep this.

Reply:
I undid the wrong change - good catch, thanks.